src/components/AddItemForm/AddItemForm.js

```diff
@@ -71,8 +69,8 @@ const AddItemForm = ({ listId }) => {
   const normalizeInput = (item) => {
     return item
       .toLowerCase()
-      .replace(/[.,/#!$%^&*;:{}=\-_`~()@[\]|<>+'"/?]/g, '') // remove punctuation
-      .replace(/\s{2,}/g, ' ') // remove extra spaces from removing punctuation
+      .replace(/\W/g, '') // remove non-word characters aka remove punctuation
+      .replace(/ /g, '') // remove spaces
       .trim(); // remove leading and trailing whitespace (trim must be last)
```

Comment:
Per @anderswift 's comment on this, I think the second line here might be unnecessary. When I commented it out and tested it, the spaces were still removed. Are there different cases this line covers we aren't seeing?

Reply:
Good catches. After some late-ish nights, this is why it's helpful to have a fresh set of eyes :) removing `/ / g` and the `trim()` work. See commit here: https://github.com/the-collab-lab/tcl-26-smart-shopping-list/pull/28/commits/4c0c089217464ae2967835fd0fecc2ffb9a3935a